WooCommerce/Classes/ViewModels/ProductDetailsViewModel.swift

```diff
@@ -515,6 +546,24 @@ extension ProductDetailsViewModel {
         cell.bodyLabel?.text = bodyText
     }
 
+    /// Purchase Note cell.
+    ///
+    func configurePurchaseNote(_ cell: ReadMoreTableViewCell) {
+        cell.titleLabel?.text = NSLocalizedString("Purchase note",
+                                                  comment: "Product Details > Purchase Details > Purchase note cell title")
+
+        cell.bodyLabel?.text = cleanedPurchaseNote
+
+        let readMoreTitle = NSLocalizedString("Read more",
+                                              comment: "Read more of the purchase note. Only the first two lines of text are displayed.")
+        readMoreExpanded = false
```

Comment:
I am not sure what this flag does. It seems like it is initialised to false, and then set to false here as well, but I can not find anywhere else where the value is set or read. I was wondering if it's actually necessary?

Reply:
Good eye! That's an artifact from trying out WordPressUI's "expand selected row" method in a previous commit. Looks like I missed some pieces after trying it out.